jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/ViewResourceDialog.kt

```diff
@@ -14,9 +15,9 @@ class ViewResourceDialog(project: Project, val resourceType: String, actionTitle
     private val component by lazy {
         panel {
             row("$resourceType:") {
-                textField(::resourceName).constraints(grow).withErrorOnApplyIf("$resourceType must be entered") {
+                textField(::resourceName).withErrorOnApplyIf("$resourceType must be entered") {
                     it.text.isNullOrBlank() || checkResourceNameValidity(it.text)
-                }
+                }.constraints(grow).growPolicy(GrowPolicy.SHORT_TEXT)
```

Comment:
personally would have formatted like this:
```suggestion
                textField(::resourceName)
                    .constraints(grow)
                    .growPolicy(GrowPolicy.SHORT_TEXT)
                    .withErrorOnApplyIf("$resourceType must be entered") {
                        it.text.isNullOrBlank() || checkResourceNameValidity(it.text)
                    }
```

Reply:
also i don't think `.constraints(grow)` is needed since the "grow policy" (bad name) already sets the width of the field

Reply:
Reformatted and removed constraints